linera-core/src/updater.rs

```diff
@@ -385,10 +385,9 @@ where
                 }
             }
         }
-        let query = ChainInfoQuery::new(chain_id).with_manager_values();
```

Comment:
Do we still need `.with_manager_values()`?

Reply:
It's still used in `try_synchronize_chain_state_from`.